catalog/imaging/imaging-plugin-nitf/src/test/java/org/codice/alliance/plugin/nitf/NitfRenderPluginTest.java

```diff
@@ -0,0 +1,510 @@
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.alliance.plugin.nitf;
+
+import static org.codice.alliance.plugin.nitf.NitfRenderPlugin.ORIGINAL;
+import static org.codice.alliance.plugin.nitf.NitfRenderPlugin.OVERVIEW;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.text.ParseException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.codice.ddf.catalog.async.data.api.internal.ProcessCreateItem;
+import org.codice.ddf.catalog.async.data.api.internal.ProcessDeleteItem;
+import org.codice.ddf.catalog.async.data.api.internal.ProcessItem;
+import org.codice.ddf.catalog.async.data.api.internal.ProcessRequest;
+import org.codice.ddf.catalog.async.data.api.internal.ProcessResource;
+import org.codice.ddf.catalog.async.data.api.internal.ProcessUpdateItem;
+import org.codice.imaging.nitf.core.common.NitfFormatException;
+import org.codice.imaging.nitf.core.image.ImageSegment;
+import org.codice.imaging.nitf.render.NitfRenderer;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+
+import ddf.catalog.data.Attribute;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.types.Core;
+import ddf.catalog.plugin.PluginExecutionException;
+
+public class NitfRenderPluginTest {
+
+    public static final String TEST_ID = "101ABC";
+
+    private ArgumentCaptor<Attribute> attributeArgumentCaptor =
+            ArgumentCaptor.forClass(Attribute.class);
+
+    NitfRenderPlugin nitfRenderPlugin;
+
+    @Before
+    public void setUp() {
+        nitfRenderPlugin = new NitfRenderPlugin();
+    }
+
+    @Test
+    public void testRunTimeException()
+            throws IOException, PluginExecutionException, ParseException, NitfFormatException {
+        // given:
+        NitfRenderer nitfRenderer = mock(NitfRenderer.class);
+        when(nitfRenderer.render(any(ImageSegment.class))).thenThrow(RuntimeException.class);
+        NitfRenderPlugin nitfRenderPlugin = new NitfRenderPlugin() {
```

Comment:
Nit: Do we need to redeclare this `NitfRenderPlugin`, just use the one previously declared on line 60?